docker-compose-dev.yml

```diff
@@ -5,7 +5,7 @@ services:
     build:
       context: .
       dockerfile: docker/Dockerfile
-    image: openpath/opdash:0.0.1
+    image: op-admin-dash-dev/opdash:0.0.1
```

Comment:
There is no such organization either.
Put it in `e-mission` or in `shankari`

Reply:
solved in https://github.com/e-mission/op-admin-dashboard/pull/7/commits/62307f87190e87e9ee7e390b41681120d79f613f